index.html

```diff
@@ -164,6 +164,30 @@ <h3>Skills</h3>
                     <div class="bulb"></div>
                 </div>
             </div>
+        </section>
+    <section class="section-4">
+        <div class="mobile-form-wrapper">
+            <h3 class="get-started">Get Started</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus architecto, atque laudantium autem vel explicabo soluta quos fugit quod distinctio. Ea ullam ut facere numquam minima dignissimos? Mollitia, esse iure.</p>
+        <form action="https://formspree.io/f/xoqzvkpg" method="post">
+            <label for="Name"><input type="text" placeholder="Your name" maxlength="30" class="name" required></label>
+            <label for="Email"><input type="email" placeholder="Your email" class="email"required></label>
+            <label for="Message"><textarea type="text" name="" maxlength="500" class="message" placeholder="Enter your message here.." required></textarea></label>
```

Comment:
Please make sure you added name attribute correctly. So needs to be "message" here. Otherwise your contact form will not submit the data.